pages/docs/customization/themes.mdx

```diff
@@ -28,21 +28,20 @@ export const meta = {
   3. Update the value of `theme.type`, and the theme of all components will follow automatically.
 
 ```jsx
-import { CSSBaseline, ZEITUIProvider, useTheme } from '@zeit-ui/react'
+import { CSSBaseline, ZEITUIProvider } from '@zeit-ui/react'
 
-const [type, setType] = useState('light')
-const currentTheme = useTheme()
-const switchThemes = () => () => {
-  const nextTheme = currentTheme.type === 'dark' ? 'light' : 'dark'
-  setType(nextTheme)
+const App = () => {
+  const [themeType, setThemeType] = useState('dark')
+  const switchThemes = () => {
+    setThemeType(themeType === 'dark' ? 'light' : 'dark')
```

Comment:
```suggestion
    setThemeType(lastThemeType => lastThemeType === 'dark' ? 'light' : 'dark')
```
Can we remove `themeType`?

Reply:
Ay! Thought this wouldn't work initially, but it does xD